src/ServiceControl/Bootstrapper.cs

```diff
@@ -221,5 +216,14 @@ private void RecordStartup(LoggingSettings loggingSettings, EndpointConfiguratio
         private TransportSettings transportSettings;
         TransportCustomization transportCustomization;
         private static HttpClient httpClient;
+
+        public async void Dispose()
```

Comment:
Should i be calling Stop in the dispose?  Also because i'm calling an async method i had to add async to the dispose method signature.  Is this even valid?

Reply:
It is a valid signature but very dangerous because once the first await statement is reached for the caller calling dispose the call is over and the appdomain would shut down